### DISCUSSION TITLE
Explorer Patcher. Compatability with 23h2

### DISCUSSION THREAD

**Discussion by Kinnyr90** [valinet/ExplorerPatcher#2401] [Q&A]
Hi,

I'm new to this site.

I have windows 11 pro 64 bit running on a desktop pc custom build

I have used explorer patcher in the past on windows 11 and it has crashed my system.

More specific after I would install explorer patcher and then restart windows 11. on past 22 h2 with different build numbers,

Once it gets just to where you would see the windows 11 desktop all I would get is a black screen. Because windows file explorer is no longer installed or missing. So what I would have to do is hit control alt delete and get to the task manger and then hit the rtart from there and then press and hold the shift key to get to the blue troubleshooting screen or repair screen. and there would be an option in there to unistall the latest update. Once I did that and then rebooted everything was fine again.

But the latest update at that time was not a major Update Like 23h2 for example. So it didn't revert windows 11 back to a previos version it just uninstalled the latest cumulative update. Leaving windows 11 in tact with all of my settings.

So now Saturday November 4th 2023 I have windows 11 pro 64 bit (23h2) Build (22631.2506) And in the disk cleanup app of windows 11 I already deleted the previous windows installation files. 

So my question is? Is Explorer Patcher compatable with this version of windows 11  (23h2)Build  (22631.2506) I Dont want to take the chance of installing it and then having something happen and I might even need to reinstall windows. Since the previous windows installation files are deleted..

Thank you!!

**Comment by pyrates999** [comment]
That is a preview update you got installed: https://support.microsoft.com/en-us/topic/october-31-2023-kb5031455-os-builds-22621-2506-and-22631-2506-preview-6513c5ec-c5a2-4aaf-97f5-44c13d29e0d4

Uninstall it if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

You should only be installing preview updates if you are a developer or are OK with running pre-release versions of EP.

**Reply by Kinnyr90** [reply]
I didn't install this update from windows Update I went to here

https://www.microsoft.com/en-in/software-download/windows11 and used the Windows 11 installation assistant.

**Reply by pyrates999** [reply]
Check if symbols were downloaded: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

You can check using the console in explorer patcher.  Open up the properties application for explorer patcher, go to advanced, and click on enable console.

Post the contents of the console here.

**Comment by Kinnyr90** [comment]
I don't have Explorer Patcher Installed yet I'm asking if the latest version of explorer Patcher is safe to download with my version of windows 11? I don't want it to crash anything Like File explorer? I also don't understand how or why Windows 11 is showing a preview Build Number when I never took 23h2 from windows update directly I got it from the Link I sent you. So I do not have explorer patcher Installed yet. Can you send me the link to the latest version of Explorer Patcher? And the preview Build is just a preview of what's in 23h2. But your version would still be showing 22h2 and Mine isn't it's showing 23h2 in my system info. So I don't understand this.

**Reply by pyrates999** [reply]
Here's the latest release: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.2428.59.1_a7c87ce
But since you have a preview update installed, you could also try the latest pre-release version: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.2428.59.1_ac268b1

**Reply by milsabords** [reply]
Latest pre-release works fine on my 23H2 .

**Reply by Kinnyr90** [reply]
Okay Thank you so much for taking the time to answer me I can be a handful sometimes. I have disabilities. I know we are good as they say.

**Reply by Kinnyr90** [reply]
See. The way I understand things about this particular preview build in question is that. When you download it from windows update. Is that it install just the Features of 23h2. But your system does not show your windows version as 23H2 It still shows it in your system specs as 22h2 and maybe the build number I think changes But your version number does not change from 22H2 TO 23H2 Irt stays at 22H2. and Mine has changed to 23H2. So I'm confused. Please help. And as I have said I didn't take the final release of 23H2 From windows update I never saw a final release so I forced the Update by downloading it from Microsofts website. So I'm not sure if I'm really on a preview version or not. I know what the build says. Please help me make sense of all of this.

**Reply by milsabords** [reply]
Same here, I upgraded using the 23H2 ISO. WU would not offer it to me, as my processor is not supported. 
After that I had several updates from WU. I do'nt see any new feature compared to 22H2. There is a cumulative preview in the pipe, I wait until it's final, maybe next patch tuesday.

**Reply by pyrates999** [reply]
So you did download the preview update and install it?

**Reply by milsabords** [reply]
This is the official release ISO,  from https://www.microsoft.com/en-us/software-download/windows11
In fact, the install of the preview offered by Windows Update failed on my PC.

![23H2](https://github.com/valinet/ExplorerPatcher/assets/45862117/2d8a47c3-3872-412a-8346-bd837304ef17)

**Reply by pyrates999** [reply]
First try the last release of EP and if that doesn't work, install the pre-release version if it is newer then the release version.

**Reply by Kinnyr90** [reply]
yes I downloaded the preview and I installed it from windows update on my windows 11 desktop. My version was 22h2 and then when I downloaded the preview version it stayed at 22h2. Because I'm assuming hence the name preview version it just allows you to preview the features that come with 23h2 and of course use the new features Like Home and co pilot etc. But it does not change the version to 23h2 It might have changed the build Number I'm not sure as I can't remember the build that it changed too.  So because the preview Build In my windows Update did not change the version of windows 11 to 23H2. Thats when I decided to force the update. by  going to Microsofts site and download 23h2. I did not download ISO I did the Installation assistant which is a straight download.. So I still don't understand why My build still sghows as a preview Build.  Becase I know that the version of windows 11 on Microsofts website I believe anyways is the final release of 23h2. Unless the final release is only if your doing an ISO Then ther version is not a preview version anymore. But at this point I'm not going to start all over again.. So I will keep what I have for a version number and download the latest version of Explorer Patcher. And then try the pre version of Explorer Patcher.

**Comment by Kinnyr90** [comment]
I downloaded the preview version of EP 22621.2428.591 Its works Which is great. I also use classic shell for the start menu.  Some of the others might be better but I don't like The windows 10 start menu or Windows 8. With Classic shell I can go back to the windows 7 style which is used for Vista and xp as well. I like them.  so I have the pre release of EP. I can't find any significent differences between the pre release and the latest version of explorer patcher. As I was looking through the fixes. Except for the Builds of windows 11 that they have been tested on. Those are different but other than that from what I can see. the fixes between the 2 versions are exactly the same. Let me know if there are any major differences. with the latest release that might make me wanna download the latest release. Something I might have missed. I would of course want tthe version that works best. Sorry for being so long winded

**Reply by pyrates999** [reply]
Uninstall classic shell. That is no longer maintained.

The replacement is open shell.

Pre-release versions are for those who have installed preview updates or you're a developer and need to test with the preview updates installed.

**Comment by Kinnyr90** [comment]
Okay I will do that.
Thank you so much!!
    On Sunday, November 5, 2023 at 11:10:25 PM EST, pyrates999 ***@***.***> wrote:  
 
 


Uninstall classic shell. That is no longer maintained.

The replacement is open shell.

**Comment by PeaceByJesus** [comment]
All was well before and after a clean install with my customization (with  7Tt  + Open  Shell with EP  being part of this – thanks), but with some trepidation I let Windows 11 to install ver. 23h2.

Which basically prevented both Explorer Patcher (22621.2861.62.2) and thus 7Tt from working as they should (as if MS targeted), and thus I am left with the 3 options of Personalization>Toolbar>Taskbar behaviors.

In which there is an option to combine Taskbar buttons or Never, or combine “when taskbar is full,” yet, there there is zero option to include labels when combined, thus requiring mouse hover to tell what they are.

The only alternative for me was to select combine “when taskbar is full, but that is a deficient substitute for what 7Tt enabled (one defect is that if you click on a combined icon, it does not show a list but it opens the last opened one.
You have to mouse-hover and wait for the pop up to get a list – which should be vertical, as a reg hack can enable) to choose from.

Meanwhile, if “Never” combine is chosen, then you have a button for every single window and document. And with the pop up for each folder and document all showing the same items!

In conclusion, the previous work around via Explorer Patcher and 7+ Taskbar Tweaker does not work for me with W/11 ver. 2362.

I read that 23h2 broke stardock as well: https://forums.stardock.com/526280/win-23h2-update-broke-start-menu-options

Below are linked images, with the first being with EP and 7Tt running but not working to combine, and with me attempting d all 3 W/11 options under Personalization>Toolbar>Taskbar behaviors, with combine “when taskbar is full” being selected in the first image. Note that the pop up on the File explore icons is that same thing you see on any other explorer buttons.

The second linked image has at top when “Never” is chosen, and the lower one is when combine “When taskbar is full.

https://i.postimg.cc/65DrwyWk/W-23h2-TBno-group.jpg

https://i.postimg.cc/52CmF8K9/combinTBgoptions.jpg

Evidently, programmers did not conceive of users having many documents open in which they want a compact yet identifiable taskbar buttons for. However, I thank God for what all I have - and thank you for helping so many with EP.

**Reply by pyrates999** [reply]
When upgrading windows 11 from say 22h2 to 23h2, uninstall EP first.  Then when installing EP again, open up EP properties application and go to about and click on restore default settings.

As for the never combine feature, you're not the first: https://www.bleepingcomputer.com/news/microsoft/windows-11s-new-never-combine-icons-feature-is-almost-usable/

**Reply by PeaceByJesus** [reply]
Thanks: My preference is to show labels while combined, which is not an option aside from combine when TB is "full." 

Meanwhile,  I had done a reset to W/11 22h2 and I think I upgraded to 23h2 before installed EP, and later I uninstalled  EP (via C:\Windows\System32\appwiz.cpl) after the 23h2 upgrade, but the reg files are still there, such as. C:\ProgramData\Microsoft\Windows\Start Menu\Programs\ExplorerPatcher\Properties (ExplorerPatcher).lnk
C:\Windows\System32\rundll32.exe
"C:\Program Files\ExplorerPatcher\ep_gui.dll",ZZGUI

So I just deleted them and restarted Windows explorer,  yet the registry still shows them present.  Need to restart (which I seldom do)?

**Reply by pyrates999** [reply]
Delete the registry entries that EP creates.  You could also do this by running the EP properties application, go to **about**, and click on **restore default settings**.

**Reply by PeaceByJesus** [reply]
OK. I reinstalled EP, then did as you said, and so now I see nothing in the Reg for "Patcher"  except C:\Program Files\ExplorerPatcher\ep_weather_host.dll and ExplorerPatcher Weather Host. 

Taskbar looks better, with 7Tt running also  but about>restore default settings nuked a couple or so other tweaks  such as by Ultimate and Win Areo tweakers. But I should have config files for those.   The most noticeable was that the minimalist W/11 right click context   menu  was back, but I see that EP offers nuking that as an option, which worked!  And I forgot, my Quick Launch toolbar which MS eliminated is back!   Thank God  you and for options that improve and for  support!

**Reply by pyrates999** [reply]
You're welcome :)
